File: Basic Coding Problems/02 If - Elses/020 Write a program to find a maximum between three numbers/Java/maximum_between_three_numbers.java
import java.util.Scanner;

public class maximum_between_three_numbers {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter three numbers: ");
        int num1 = scanner.nextInt();
        int num2 = scanner.nextInt();
        int num3 = scanner.nextInt();

        int maxNum = num1;
        if (num2 > maxNum) {
            maxNum = num2;
        }
        if (num3 > maxNum) {
            maxNum = num3;
        }

        System.out.println("The maximum number is: " + maxNum);
    }
}

File: Basic Coding Problems/02 If - Elses/020 Write a program to find a maximum between three numbers/C/maximum_between_three_numbers.c
#include <stdio.h>

int main() {
    int num1, num2, num3;
    printf("Enter three numbers: ");
    scanf("%d %d %d", &num1, &num2, &num3);

    int max_num = num1;
    if (num2 > max_num) {
        max_num = num2;
    } if (num3 > max_num) {
        max_num = num3;
    }

    printf("%d is the maximum number\n", max_num);
    return 0;
}